docs/source/releases/next.rst

```diff
@@ -0,0 +1,45 @@
+========================
+104.0 - TBD
+========================
+
+The Avocado team is proud to present another release: Avocado 1034.0,
```

Comment:
Typo here in version 1034.0! Will we get there? :smile:

Reply:
Right, it is a little bit scary, I will fix it 😄